Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandCreatePlayerHouseKey.java

```diff
@@ -0,0 +1,55 @@
+package com.universeprojects.miniup.server.commands;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.universeprojects.cacheddatastore.CachedDatastoreService;
+import com.universeprojects.cacheddatastore.CachedEntity;
+import com.universeprojects.miniup.server.ODPDBAccess;
+import com.universeprojects.miniup.server.commands.framework.Command;
+import com.universeprojects.miniup.server.commands.framework.UserErrorMessage;
+
+public class CommandCreatePlayerHouseKey extends Command{
+
+	/**
+	 * Creates key(this is an item that the character holds) to enter his house,
+	 * 
+	 * @author Seathninja
+	 */
+	
+	public CommandCreatePlayerHouseKey(ODPDBAccess db, HttpServletRequest request, HttpServletResponse response)
+	{
+		super(db, request, response);
+	}
+	
+	public void run(Map<String, String> parameters) throws UserErrorMessage 
+	{
+		
+		//preliminary creating the item into existence
```

Comment:
No checks whatsoever if the player is allowed to make a key. Remember that people have access to the javascript and even a novice programmer can issue a request that contains whatever he wants. We need to watch out for malicious requests that try to do things they're not allowed to do. For examples, check out any one of the other commands (they pretty much all have some form of sanity checking in place).